web-components/src/signals/questions.ts

```diff
@@ -0,0 +1,74 @@
+import { Computed, State } from "@lit-labs/signals"
+import robotoff from "../api/robotoff"
+import {
+  QuestionAnnotationAnswer,
+  QuestionRequestParams,
+} from "../types/robotoff"
+
+export const questions = new State<any[]>([])
+
+/** Current question index */
+export const currentQuestionIndex = new State(0)
+/** Has the user answered to one question */
+export const hasAnswered = new State(false)
+/** Does the current product has questions */
+export const hasQuestions = new Computed(() => questions.get().length > 0)
+/** Number of questions available */
+export const numberOfQuestions = new Computed(() => questions.get().length ?? 0)
+/** Indicates if all questions have been answered */
+export const isQuestionsFinished = new State(false)
```

Comment:
I'm not sure it's a good idea to have a global state as we spoke about displaying robotoff questions at several places.